charts/fybrik/files/taxonomy/data_catalog_request.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Data Catalog Request",
+	"type": "object",
+	"properties": {
```

Comment:
should we add an operation type for the request? i.e., read, create, delete assets? 
no need to implement for now, but how would it be reflected in the request (just the type for now).

Reply:
Right, we'll need the correct credentials based on the operation. Today, we get only "read" credentials.

Reply:
added `operationType` field for this. Please review.

Reply:
@rohithdv I think the we need concerete operation types. we can start with "read" or "get", but expect we would need "create, "update" and "delete" in the future.
Since at this time only the manager talks with the catalog connector, it should be in the base taxonomy.

Reply:
@ronenkat I don't understand your goal. This json schema is for the get dataset details call. My understanding from the other comment is that operation type is for the get asset scenario, but required for knowing which credential permissions to obtain.

If your aim is to be future proof for other API calls like creating and deleting an asset in the catalog then I agree that we should factor out common definitions to a different schema. My suggestion is to create `data_catalog.json` with common definitions related to catalog assets and refer to that file from different request/response schemas.

Reply:
As discussed, moved commond definitions of `details`, `credentials` and `resource` to `data_catalog.json` and referred to them in `data_catalog_response.json`. @ronenkat @roee88